### PR TITLE
Remove config setting raw output

### PR DESCRIPTION
**Pull request by renaissanceGeek**

### CODE REVIEW

File: cmd/cubic-cli/cmd/transcribe.go [hunk 1]
```diff
@@ -395,7 +395,7 @@ func transcribeFiles(workerID int, wg *sync.WaitGroup, client *cubic.Client,
 		case ".mp3":
 			audioEncoding = cubicpb.RecognitionConfig_MP3
 		case ".vox":
-			audioEncoding = cubicpb.RecognitionConfig_VOX8000
+			audioEncoding = cubicpb.RecognitionConfig_ULAW8000
```
Comment (jhollowayj) [inline cmd/cubic-cli/cmd/transcribe.go:400]: So, how to we trigger `RecognitionConfig_VOX8000`?

**Review by jhollowayj — CHANGES_REQUESTED** [review]
I'm fine with the cfg.EnableRawTranscript change.

However, I don't think that the vox->ulaw change will be misleading to new users of this tool.

File: cmd/cubic-cli/cmd/transcribe.go (hunk 1 above)
Comment (renaissanceGeek) [reply cmd/cubic-cli/cmd/transcribe.go:400]: We can't yet in this release.  So far the only customer who has asked for .vox file wants mu-law.  Alok's suggestion was that we would have configuration that they can override to specify mu-law for the vox extension, but we haven't implemented that yet.

Comment (renaissanceGeek) [reply cmd/cubic-cli/cmd/transcribe.go:400]: ADPCM may be most common, but mu-law and a-law are also valid vox encodings

**Review by jhollowayj — APPROVED** [review]

**Pull request merged**